tests/optim/test_oss.py

```diff
@@ -726,11 +766,17 @@ def check_optimizer_equivalence(optimizer: Type[torch.optim.Optimizer]):
         model.register_buffer("test_buffer", torch.ones((1)) * rank)
         model.to(device)
 
-        sharded_optimizer = optim.OSS(params=model.parameters(), optim=optimizer, lr=1e-3)
+        optimizer_settings = {"lr": 1e-3}
```

Comment:
@msbaines I can add another group in this ddp parity test, let me know if you had something else in mind